cmd/finch/main.go

```diff
@@ -117,7 +117,7 @@ func virtualMachineCommands(
 		logger,
 		optionalDepGroups,
 		config.NewLimaApplier(fc, fs, fp.LimaOverrideConfigPath()),
-		config.NewNerdctlApplier(fssh.NewDialer(), fs, fp.LimaSSHPrivateKeyPath()),
+		config.NewNerdctlApplier(fssh.NewDialer(), fs, fp.LimaSSHPrivateKeyPath(), system.NewStdLib().Env("USER")),
```

Comment:
Maybe refer to the previous rootful change. https://github.com/runfinch/finch/commit/ae43a0779c778cc3656724f84b5cdfd8ab087798#diff-531afb9f97a7384e0de895046f9e14befbebc49c896b080fce4c41c75f14f1b4

The user was populated by `user := nca.systemDeps.Env("USER")`

Reply:
It's the same thing except we only pass the USER here as we only need it in NewNerdctlApplier

Reply:
In the previous way, the variable is input from main.go.

Reply:
Could you please elaborate a bit? This variable is also input from main.go

Reply:
Yeah you are right.